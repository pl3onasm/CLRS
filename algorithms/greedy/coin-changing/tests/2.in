25 20 10 5 1.

40 67 19 28 45 81.