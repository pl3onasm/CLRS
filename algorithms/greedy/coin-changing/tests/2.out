Non-canonical coin system
... using DP approach

40 = 20 x 2
67 = 25 + 20 x 2 + 1 x 2
19 = 10 + 5 + 1 x 4
28 = 25 + 1 x 3
45 = 25 + 20
81 = 25 x 3 + 5 + 1
